src/components/ContentBoxes.tsx

```diff
@@ -14,14 +17,16 @@ const Block = ({ block }: { block: Block }): JSX.Element => (
           display: 'flex',
           alignItems: 'center',
           justifyContent: 'center',
-          width: '70px',
-          height: '70px',
+          width: iconBubbleSize + 'px',
+          height: iconBubbleSize + 'px',
           marginBottom: suomifiTheme.spacing.l,
           border: `1px solid ${suomifiTheme.colors.depthLight13}`,
           borderRadius: '50%',
-          fontSize: '50px'
+          fontSize: iconsize + 'px'
         }}>
-        <Icon icon={block.icon} />
+        <div style={{ marginBottom: '-0.1em' }}>
```

Comment:
Should we fix this in the ui-components repo? For some reason there is a transform: translateY(0.1em); defined in Icon.baseStyles.tsx, not sure why. Maybe @J-Kallunki knows?

Reply:
As discussed elsewhere, I'd propose to target the problematic style directly to icon with e.g. transform:none and remove it once this issue has been resolved in ui-components repo.